repos/rust-lang/rustup.toml

```diff
@@ -0,0 +1,11 @@
+org = "rust-lang"
+name = "rustup"
+description = "The Rust toolchain installer"
+bots = ["rustbot"]
+
+[access.teams]
+rustup = "maintain"
+mods = "maintain"
```

Comment:
I don't think we need this. The mods have org wide permissions to do what they need to do.

Reply:
Perfect!